appendices/feature_index.txt

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021 Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+// The asciidoc [glossary] template cannot contain subsections.
+// The abbreviations and prefixes probably belong in the upcoming
+// API/extension-writing-guidelines appendix, anyway.
+
+[appendix]
+[[feature_index]]
+== Feature Index
+
+* <<api_types_lights, `KHR_AREA_LIGHTS`>>: Additional lighting parameters are used to enable soft shadows
```

Comment:
Additional parameters are supported to spatially extend lights and thus potentially enable soft shadows

Reply:
Addressed in 53193bc.